components/Navbar/MoreMenu/index.tsx

```diff
@@ -0,0 +1,164 @@
+import React, { useState, useCallback } from 'react';
+import { useTranslation } from 'react-i18next';
+import { useTheme, Box, Typography, popoverClasses, Menu, MenuItem, Button } from '@mui/material';
+import Link from 'next/link';
+import MoreHorizRoundedIcon from '@mui/icons-material/MoreHorizRounded';
+import ExpandLessIcon from '@mui/icons-material/ExpandLessRounded';
+import ExpandMoreIcon from '@mui/icons-material/ExpandMoreRounded';
+
+import useRouter from 'hooks/useRouter';
+
+type Props = {
+  options: {
+    pathname: string;
+    name: string;
+    icon: React.ReactElement;
+    isNew?: boolean;
+  }[];
+};
+
+function MoreMenu({ options }: Props) {
+  const { t } = useTranslation();
+  const { pathname: currentPathname, query } = useRouter();
+  const [anchorEl, setAnchorEl] = useState<null | HTMLElement>(null);
+  const [currTimeout, setCurrTimeout] = useState<ReturnType<typeof setTimeout>>();
+  const { palette } = useTheme();
+
+  const openMenu = useCallback(
+    (event: React.MouseEvent<HTMLButtonElement>) => {
+      if (anchorEl !== event.currentTarget) {
+        setAnchorEl(event.currentTarget);
+      }
+    },
+    [anchorEl],
+  );
+  const closeMenu = useCallback(() => setAnchorEl(null), [setAnchorEl]);
+
+  const handleOnHover = useCallback(() => clearTimeout(currTimeout), [currTimeout]);
```

Comment:
```suggestion
  const handleHover = useCallback(() => clearTimeout(currTimeout), [currTimeout]);
```

Reply:
same for the other one, this is for consistency but not very important, skip if needed